pyquil/tests/test_operator_estimation.py

```diff
@@ -29,19 +40,19 @@ def _generate_random_paulis(n_qubits, n_terms):
     return operators
 
 
-def test_experiment():
-    in_ops = _generate_random_paulis(n_qubits=4, n_terms=7)
+def test_experiment_setting():
+    in_states = _generate_random_states(n_qubits=4, n_terms=7)
     out_ops = _generate_random_paulis(n_qubits=4, n_terms=7)
-    for iop, oop in zip(in_ops, out_ops):
-        expt = ExperimentSetting(iop, oop)
+    for ist, oop in zip(in_states, out_ops):
+        expt = ExperimentSetting(ist, oop)
         assert str(expt) == expt.serializable()
         expt2 = ExperimentSetting.from_str(str(expt))
         assert expt == expt2
-        assert expt2.in_operator == iop
+        assert expt2.in_state == ist
         assert expt2.out_operator == oop
 
 
-def test_experiment_no_in():
+def test_setting_no_in_back_compat():
```

Comment:
This test as well as `test_setting_no_in(..)` both raise the warning message `UserWarning: ExperimentSetting.in_operator is deprecated in favor of in_state`. In general, we should try to avoid writing tests that will raise warning messages. If we add the decorator `@pytest.mark.filterwarnings("ignore:ExperimentSetting")` to both these tests, we'd have a warning free test output (see https://docs.pytest.org/en/latest/warnings.html)